### ISSUE TITLE
Microsoft Learn Great ,But ! 

### ISSUE THREAD

**Issue by h0s** [MicrosoftDocs/feedback#705]
**Is your feature request related to a problem? Please describe.**
No
**Describe the solution you'd like**
I started learning with the New Microsoft Learn and discovering different area, is really very nice interactive and let you learn more and more without get boring and especially let me take the challenge on Azure, but as a .Net /Xamarin developer I would like to find more content about other technologies than Azure such as.net core and if there is any system like challenges or a page where we find top learner depend on their collected points (like the old TechRewards program).

Thank you.

**Comment by lnyswonger** [comment]
Thank you for your feedback! At Ignite 2018, we delivered role-based training and learning paths for Azure Developer, Azure Solution Architect, and Azure Administrator, as well as role-based training and learning paths for Power BI, PowerApps, Microsoft Flow, and Dynamics 365 roles. Post-Ignite, we will continue to build content beyond these areas and improve the overall experience with additional features than what you see today.

**Comment by Powerhelmsman** [comment]
@h0s Thanks for your feedback. I added an internal work item to track it. Once we add it, will let you know.

**Comment by anthcool** [comment]
I agree with @h0s that there are those of us looking forward to different content than Azure at some point.  I, too, am doing Xamarin and .NET Core development and would love how-tos & whys, etc. on building Xamarin apps as well as architecture choices. Also, understanding APIs, async/await use, best practices for ASP.NET Core Web API integration, etc.  C# in general.  Is this MSFT Learn initiative to replace MVA at some point?

**Comment by Powerhelmsman** [comment]
@yunyunyunyun Assign this feedback to you since you are Learn PM.

**Comment by Regenhardt** [comment]
So learn has been here for a while, MVA has been gone for some time now. There is another apprentice in our company now, and there's still most of the MVA content missing from MS Learn. I'm honestly giving people the youtube playlists that saved the MVA videos. Where's the content MSFT kept talking about?
